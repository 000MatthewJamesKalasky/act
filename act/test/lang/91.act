defproc testme (chan(int) l, r)
{
  int x;

  chp-txt {
    while  {
   case true:
    select {
     case x > 0 :  l?x; , r!x;
     case x < 0 :  r!x;
     else : x := x + 1;
     }
     wait-for (x > 0);
    }
  }
}

testme t;
