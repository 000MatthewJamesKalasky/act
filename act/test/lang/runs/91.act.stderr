In expanding testme (91.act:1)
In expanding ::<Global>
Error on or near line number 5.
WARNING: potential write conflict in CHP description
   Variable: `x'
